config/routes.rb

```diff
@@ -1,6 +1,15 @@
 Rails.application.routes.draw do
   devise_for :users
-  root 'splash_screen#index'
+
+  devise_scope :user do
+    authenticated :user do
+      root 'categories#index', as: :authenticated_root
```

Comment:
- [ ] Here you made the home page for categories the root but that is not the case for the live version. Instead, I go to the page with a button for categories and log out. 
![image](https://user-images.githubusercontent.com/45460192/220407486-b6f9c1b4-4ef5-4eb1-be78-ce55ac666362.png)